olp-cpp-sdk-dataservice-read/src/generated/api/StreamApi.h

```diff
@@ -256,6 +256,9 @@ class StreamApi {
       const boost::optional<std::string>& mode,
       const client::CancellationContext& context, const std::string& endpoint,
       std::string& x_correlation_id);
+
+  static bool HandleCorrelationId(const http::HeadersType& headers,
```

Comment:
Private static methods feel more comfortable of they are anonymous functions :) Keeps the headers clean.